### PR TITLE
Small Dawn Fixes

### PR DESCRIPTION
**Pull request by acpaquette**
Fixed some imports with dawn to get things working

### CODE REVIEW

File: ale/drivers/dawn_drivers.py [hunk 1]
```diff
@@ -45,6 +47,7 @@ def metakernel(self):
         metakernel_dir = config.dawn
         mks = sorted(glob(os.path.join(metakernel_dir,'*.tm')))
         if not hasattr(self, '_metakernel'):
+            self._metakernel = None
             for mk in mks:
                 if str(self.start_time.year) in os.path.basename(mk):
```
Comment (paarongiroux) [inline ale/drivers/dawn_drivers.py]: should be utc_start_time

File: ale/drivers/dawn_drivers.py [hunk 2]
```diff
@@ -223,3 +227,38 @@ def sensor_model_version(self):
...
+        Returns
+        -------
+        : float
+          center detector line
+        """
+        return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[1])
+
+
+    @property
+    def detector_start_sample(self):
```
Comment (acpaquette) [inline ale/drivers/dawn_drivers.py]: I'm not sure if detector_start_sample, or detector_start_line is correct. Was the closest thing I could find in the dawn ik kernels that looked similar to messenger.

**Comment by acpaquette** [comment]
This PR is dependent on #201 for the rotation hotfix

File: ale/drivers/dawn_drivers.py (hunk 2 above)
Comment (acpaquette) [reply ale/drivers/dawn_drivers.py]: I think they actually are just both 1?

**Pull request merged**